vta/python/vta/top/graphpack.py

```diff
@@ -0,0 +1,308 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Let's rename this file.

Reply:
Do you suggest that we rename it to bitpack instead?